server/README.md

```diff
@@ -48,9 +49,33 @@ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 sudo apt install make gcc pkg-config libavahi-compat-libdnssd-dev libpq-dev
 ```
 
-# Run without mSupply central
+# Set up sync with mSupply central
+
+Remote server data is configured through mSupply central server, when the app first starts it's expected to initialise from mSupply.
+
+### Set up omSupply remote site in mSupply
+
+- Ensure you have mSupply set up locally
+- In Admin > Preferences > General, ensure `Synchronisation is active` is selected
+- In Special > Synchronisation
+  - Open (or create) the site for our Open mSupply instance
+  - Does it have a hardware ID already? If so, refresh the hardware ID so it doesn't have one.
+- Initialise your omSupply instance - you can do this two ways:
+
+  - Via YAML configuration:
+    - In your Open mSupply repo, under `server/configuration` copy the `example.yaml` to `local.yaml`, and uncomment the contents of the file. Under the `sync` config, ensure the `username` is set to the site name from mSupply (i.e. `remote`)
+    - Start the server: from the `server` root, run `cargo run`
+    - This should sync with your mSupply instance, and make the omSupply server available for local use
+  - Via the UI (this may be the easier option when you are switching between different APIs):
+    - Start the server: from the `server` root, run `cargo run`
+    - Start the UI: from the `client` root, run `yarn start-local`
+    - You should see an initialisation screen - enter the site details from mSupply
```

Comment:
Nit: Could this be moved below since kinda repeating...

```
Now we can start server with

cargo run
```

with the above

then...

```NOTE: make sure that sync configurations in configuration/*.yaml file is commented out if running without mSupply central, otherwise may get an error that database and yaml sync configurations differ (in which case remote server will try to contact central server)```